src/node_util.cc

```diff
@@ -21,17 +21,20 @@ static void GetOwnNonIndexProperties(
   Environment* env = Environment::GetCurrent(args);
   Local<Context> context = env->context();
 
-  if (!args[0]->IsObject())
+  if (!args[0]->IsObject() || !args[1]->IsUint32())
```

Comment:
Wouldn't it make sense for this to be a `CHECK` since we should never end up here without said args?

(Not critical for this PR)